Services/Models/Simulation.cs

```diff
@@ -60,7 +60,7 @@ public bool IsActiveNow
         public bool DevicesCreationComplete { get; set; }
 
         [JsonProperty(Order = 13)]
-        public bool DevicesCleanUpRequiredByUser { get; set; }
+        public bool DeleteDevicesWhenSimulationEnds { get; set; }
```

Comment:
please make this explicitly `false` by default in the ctor (line 135)

Reply:
Fixed.